vignettes/opencl-files/bernoulli_logit_glm.stan

```diff
@@ -0,0 +1,15 @@
+data {
+  int<lower=1> k;
+  int<lower=0> n;
+  matrix[n, k] X;
+  int y[n];
+}
+parameters {
+  vector[k] beta;
+  real alpha;
+}
+model {
+  target += std_normal_lpdf(beta);
+  target += std_normal_lpdf(alpha);
+  target += bernoulli_logit_glm_lpmf(y | X, alpha, beta);
+}
```

Comment:
I could just use the profiling GLM model, the path to the model would be "profiling-files/bernoulli_logit_glm.stan". Is that potentially confusing? I guess a few lines of Stan code is not a potential problem size-wise?

Reply:
It is probably just as advantageous to have this self-contained.